tests/test_other.py

```diff
@@ -8027,12 +8027,6 @@ def test_binaryen_metadce_cxx_fastcomp(self, *args):
     'export_nothing':
           (['-Os', '-s', 'EXPORTED_FUNCTIONS=[]'],
                      0, [], [],          61,  0,   1,  1), # noqa
-    # we don't metadce with linkable code! other modules may want stuff
-    # don't compare the # of functions in a main module, which changes a lot
-    # TODO(sbc): Investivate why the number of exports is order of magnitude
-    # larger for wasm backend.
-    'main_module_1': (['-O3', '-s', 'MAIN_MODULE=1'], 1610, [], [], 517336, 172, 1507, None), # noqa
-    'main_module_2': (['-O3', '-s', 'MAIN_MODULE=2'],   15, [], [],  10770,  17,   13, None), # noqa
```

Comment:
Should we leave the main_module_2 in there since that is supposed to be minimal?

Reply:
Ah, very good point - I'll restore those.